app/views/sitemaps/show.xml.erb

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+  <%= @articles.each do |article_obj| %>
```

Comment:
If I'm not mistaken here, `article_object` is a two element array, so maybe de-structure it in the block argument to make the rest of the code more self-documenting:

```ruby
<%= @articles.each do |path, last_comment_at| %>
  <url>
    <loc><%= app_protocol_and_domain %><%= path %></loc>
    <lastmod><%= last_comment_at %></lastmod>
  </url>
<% end %>
```

Reply:
👍